_elements/buttons.md

```diff
@@ -70,7 +70,11 @@ lead: Use buttons to signal actions.
   </button>
   <div id="collapsible-0" aria-hidden="false" class="usa-accordion-content">
     <h4 class="usa-heading">Accessibility</h4>
-    <p>No specific accessibility guidelines.</p>
+    <ul>
+      <li>Buttons should display a visible focus state when users tab to them.</li>
+      <li>Whenever possible, use the <code>&lt;button&gt;</code> to display buttons rather than styling links to look like buttons. Screen readers treat buttons differently than they do links.</li>
```

Comment:
Is this accurate? I think it's perfectly acceptable to style links like buttons. Buttons don't have `href` properties, they are more for actionable forms.